src/documents/ComponentConfigurationOptions.md

```diff
@@ -179,80 +195,51 @@ The following options are supported by all component grades:
   </tr>
 </table>
 
-### dynamicComponents ###
+### `distributeOptions` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object containing named definitions of the component's dynamic subcomponents</td>
-  </tr>
-  <tr>
-    <th>Notes</th>
-    <td>Some special context names may be available within the subcomponent's definition block, for example <code>{source}</code> and <code>{sourcePath}</code> or <code>{arguments}</code>. This framework facility will be replaced by a more declarative equivalent in time and should be used with caution.</td>
+    <td>A record directing the framework to distribute options from this component to one or more other components in the component tree. Either a single, an <code>Array</code> or <code>Object</code> holding these records is supported.
+    In the <code>Object</code> form, the keys of the object will be taken to represent the <code>namespace</code of the distribution.</td>
   </tr>
   <tr>
     <th>Example Definition</th>
     <td><pre>
 <code>fluid.defaults("component.name", {
-    dynamicComponents: {
-        dynamic1: {
-            type: "component.subcomp1",
-            source: "{context}.someArray",
-            options: {...}
-        },
-        ...
-    },
+    gradeNames: ["fluid.component"],
+    distributeOptions: {
+        namespace: "myDistribution",
+        record: "another.grade.name",
+        target: "{that target.grade.name}.options.gradeNames"
+    }
     ...
 });</code>
 </pre></td>
   </tr>
   <tr>
     <th>See also</th>
-    <td><a href="tutorial-gettingStartedWithInfusion/Subcomponents.md">Tutorial: Subcomponents</a></td>
+    <td><a href="IoCSS.md">IoCSS for options distributions</a></td>
   </tr>
 </table>
 
-
-## Little Components ##
-
-Components defined with a grade of `littleComponent` support all of the [common options](#options-supported-by-all-components-grades) described above, and no others. Component developers are free to define their own additional options.
-
-See also: [Component Grades](ComponentGrades.md)
-
-## Model Components ##
-
-Components defined with a grade of `modelComponent/modelRelayComponent` support all of the [common options](#options-supported-by-all-components-grades) described above, as well as those defined below. Component developers are free to define their own additional options.
-
-See also: [Component Grades](ComponentGrades.md)
-
-### model ###
+### `mergePolicy` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>An object containing the data model to be used by the component.</td>
+    <td>An object providing instructions for how particular options should be merged when integrator options are merged with default values.</td>
```

Comment:
In other parts of the documentation, we refer to "a record" rather than "an object." Is there a reason we don't do so here?